sdk/core/Azure.Core.TestFramework/src/TestLogger.cs

```diff
@@ -56,15 +51,16 @@ internal class TestLogger : IDisposable
             _log = new StringBuilder();
 
         /// <summary>
-        /// Output the Events to the console in the case of test failure.
+        /// Output the Events to the NUnit TestContext. For failed tests,
+        /// the output will be available in the individual tests attachment in
+        /// Azure Dev Ops. For passed tests, you will need to download the parent test run
+        /// attachments which will give you the trx files for all tests. These files
+        /// can be opened in VS to review the logs.
         /// This will include the HTTP requests and responses.
         /// </summary>
         public void OutputEventsForTest()
         {
-            if (_verbose || TestContext.CurrentContext.Result.Outcome.Status == TestStatus.Failed)
-            {
-                TestContext.Out.WriteLine(_log.ToString());
-            }
+            TestContext.Out.WriteLine(_log.ToString());
```

Comment:
Why the change to log everything?  Did the DevOps experience here get a radical improvement?

Reply:
I'm not sure if DevOps changed, but the passed tests will not be included in the console or in individual test attachments. You can download all the test output from the parent test run though.

Reply:
Because for potential shared resource issues, we need successful logs to see who created / updated / deleted it that cause other tests to fail.